src/main/scala/chiseltest/simulator/ipc/VpiVerilogHarnessGenerator.scala

```diff
@@ -8,7 +8,8 @@ import chiseltest.simulator.TopmoduleInfo
 private[chiseltest] object VpiVerilogHarnessGenerator {
   def codeGen(
     toplevel:    TopmoduleInfo,
-    moduleNames: Seq[String]
+    moduleNames: Seq[String],
+    useVpdDump: Boolean = false
```

Comment:
Do you know why this isn't used? Seems like it would break for tools that do not support the vcs specific commands.